VT-LPR/README.md

```diff
@@ -0,0 +1,107 @@
+
+## Challenges
+Image based vehicle type and license plate recognition include many challenges such as:
+* Intra-class variations
+* Background clutter
+* Non-uniform position
+* Multiple license plates
+* Non-uniform fonts
+* Partial occlusion
+* Damaged license plates
+* Lack of license plates
+
+<p align="center">
+  <img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/problems.jpg" title="hover text">
+  <img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/variations.jpg" title="hover text">
+</p>
+
+## Methodology
+### Dataset
+We divide the Vehicle type and license plate recognition task into three steps: Vehicle type recognition, License plate localization and License plate character recognition. So we prepared three dataset named: _preprocessed dataset for Vehicle type recognition, lp-detect dataset for license plate location and lp-read dataset for license plate character recognition._
+As we're training separate model for each task, all the vehicle images are annotated accordingly as shown below.
+
+<img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/label.jpg" title="hover text">
+
+**Note**: As the AFAIK dataset used in this paper is proprietary and can not be downloaded anywhere, there is an alternative dataset that can be used in place and can be downloaded from <a href='https://storage.googleapis.com/openimages/web/visualizer/index.html?set=train&type=detection&c=%2Fm%2F01jfm'>here</a>.
+
+### Framework
+
+The proposed framework is aim to automatically collect toll amount by end-to-end image-based vehicle type and license plate recognition. Input to the model is an image and the outputs are recognized vehicle type and license plate character. When an image is fed to the first network called VT-Net, it recognize the vehicle type and crop the detection results. The cropped image is then fed to the secong network called LP-Net, which localize and crop the license plate. Finally, the last model called CR-Net is applied on the cropped license plate image to recognize the characters in the license plate.
+
+<p align="center">
+<img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/flow.jpg" width ="400" title="hover text">
+<img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/chart.jpg" width ="450" title="hover text">
+</p>
+
+### Results
+<p align="center">
+<img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/type_res.jpg" width ="400" title="hover text">
+<img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/lp_res.jpg" width ="320" title="hover text">
+  <img src="https://github.com/arnavrneo/VT-LPR/blob/main/Figs/lpr_res.jpg" width ="450" title="hover text">
+</p>
+
+## Test
+
+1. Download the model weights from <a href='https://dagshub.com/arnavr.neo/VT-LPR'>here</a> by ruuning `dvc pull -r origin`.
+
+2. Clone this repository
+
+``` #2x
+!git clone https://github.com/arnavrneo/VT-LPR.git
+```
+
+3. Put model weights in the downloaded folder.
```

Comment:
After you change the order between 1 and 2, you can remove 3, since it will happen as part of the `dvc pull` command